sound/soc/sof/sof-acpi-dev.c

```diff
@@ -177,35 +176,9 @@ static int sof_acpi_probe(struct platform_device *pdev)
 		return -ENODEV;
```

Comment:
ops is no longer used here. I think it is OK to remove it. The same thing I guess should happen in sof-pci-dev.c

Reply:
@dbaluta I am breaking down commits to make it easy to review. And I will handle your comments too. I'll handle your comments while I'm at it too.

Reply:
Thanks @dbaluta absolutely yes. I am only splitting commits mostly, so please do send me your patches and I'll add them to the series.